Source/WebCore/platform/graphics/coretext/FontCoreText.cpp

```diff
@@ -780,6 +780,22 @@ FloatRect Font::platformBoundsForGlyph(Glyph glyph) const
     return boundingBox;
 }
 
+Vector<FloatRect> Font::platformBoundsForGlyphs(const Vector<Glyph>& glyphs) const
+{
+    Vector<CGRect> rectsForGlyphs(glyphs.size());
+    auto ignoredBoundingBox = CTFontGetBoundingRectsForGlyphs(getCTFont(), platformData().orientation() == FontOrientation::Vertical ? kCTFontOrientationVertical : kCTFontOrientationHorizontal, glyphs.data(), rectsForGlyphs.data(), rectsForGlyphs.size());
+    (void)ignoredBoundingBox;
```

Comment:
If we still want to store `ignoredBoundingBox` I think a better approach here could be `UNUSED_PARAM`.